src/PreallocatedLinearOperators.jl

```diff
@@ -28,10 +28,14 @@ mutable struct PreallocatedLinearOperator{T} <: AbstractPreallocatedLinearOperat
   nprod :: Int
   ntprod :: Int
   nctprod :: Int
+  Mv
+  Mtu
+  Maw
```

Comment:
Is it possible to add `Mx::AbstractVector{T}` ?

Reply:
Do you mean that you want to add another vector?

Reply:
No, just parameterize `Mv`, `Mtu` and `Maw`.